### PR TITLE
Remove references to AppCache

### PR DESCRIPTION
**Pull request by argilo**
AppCache is deprecated and support has been removed from browsers. AppCache functionality has been removed from sondehub-tracker as well, so we may as well clean up the last few references to it.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -29,9 +29,6 @@
   Header append Vary User-Agent
 </IfModule>
 
-AddType text/cache-manifest .manifest
-AddType text/cache-manifest .appcache
```
Comment (TheSkorm) [inline .htaccess:33]: We should drop this entire file. it doesn't get used

Comment (argilo) [reply .htaccess:33]: True, I suppose it's leftover from a time this was once served by Apache. Deleted.

File: js/app.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ function trackerInit() {
     if(!map) load();
 }
 
-// if for some reason, applicationCache is not working, load the app after a 3s timeout
+// load the app after a 3s timeout
 var initTimer = setTimeout(trackerInit, 3000);
```
Comment (TheSkorm) [inline js/app.js:221]: Just wondering - what if we just load straight away without a timeout?

Comment (argilo) [reply js/app.js:221]: In that case, the splash screen won't be seen. Do you want to remove it?

Comment (TheSkorm) [reply js/app.js:221]: Oh, I guess we lose our sponsorship logos.... @darksidelemm there's probably little reason to wait 3 seconds at startup, but we'd a better way of doing sponsorship stuff... happy to keep it for the time being but something maybe we should think about?

Comment (TheSkorm) [reply js/app.js:221]: @argilo lets leave it in the for the time being.

Comment (argilo) [reply js/app.js:221]: If you decide to keep the splash screen, a useful improvement would be to let the rest of the app initialize behind it. At present, things like map tiles and telemetry don't even begin loading until the splash screen disappears.

**Pull request merged**